src/test/videopress/edit.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import {
+	getEditorHtml,
+	initializeEditor,
+	addBlock,
+	getBlock,
+	setupCoreBlocks,
+} from 'test/helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import { Platform } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	registerJetpackBlocks,
+	setupJetpackEditor,
+} from '../../jetpack-editor-setup';
+
+const onlyOnAndroid = Platform.select( { android: it, ios: it.skip } );
+
+const defaultProps = {
+	capabilities: {
+		videoPressBlock: true,
+	},
+};
+
+setupCoreBlocks();
+
+beforeAll( () => {
+	// Register Jetpack blocks
+	setupJetpackEditor( { blogId: 1, isJetpackActive: true } );
+	registerJetpackBlocks( defaultProps );
+} );
+
+afterAll( () => {
+	// Clean up registered blocks
+	getBlockTypes().forEach( ( block ) => {
+		unregisterBlockType( block.name );
+	} );
+} );
```

Comment:
This part is not really needed when using `setupCoreBlocks` as it already unregisters all blocks ([reference](https://github.com/WordPress/gutenberg/blob/cd25a95a972c75ec999f06e816e8bdc28035e936/test/native/integration-test-helpers/setup-core-blocks.js#L36-L41)).

Reply:
Would we still need it to unregister the Jetpack blocks?

Reply:
Apologies, slow moment from me! I looked at the linked reference and can see that all the blocks will be unregistered as part of that helper. Gone ahead to remove this cleanup in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5715/commits/635d4bcc1b096e316d6015a9b6a8ac6822ecd191.